subprojects/docs/src/docs/userguide/five_things.adoc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[five_things]]
+= Five things you need to know about Gradle
```

Comment:
How about we make it six and add build scans? :) cc @eriwen

Reply:
I think it's better to incorporate build scans where there is obvious value. They aren't necessary to use Gradle properly and inserting them here risks making it look like marketing. That said, I think we can add them to the main Getting Started page.

Reply:
I added build scans as an item under What is Gradle? in the Getting Started page.